core/src/main/java/org/axonframework/commandhandling/gateway/DefaultCommandGateway.java

```diff
@@ -143,4 +122,48 @@ public DefaultCommandGateway(CommandBus commandBus, RetryScheduler retrySchedule
         send(command, new FailureLoggingCallback<>(logger, callback));
         return callback;
     }
+
+    /**
+     * Builder class to instantiate a {@link DefaultCommandGateway}.
+     * <p>
+     * The {@code dispatchInterceptors} are defaulted to an empty list.
+     * The {@link CommandBus} is a <b>hard requirements</b> and as such should be provided.
+     */
+    public static class Builder extends AbstractCommandGateway.Builder {
+
+        @Override
+        public Builder commandBus(CommandBus commandBus) {
+            super.commandBus(commandBus);
```

Comment:
No check if not null?

As the Javadoc on the Builder class states "The {@link CommandBus} is a <b>hard requirements</b> and as such should be provided."

Reply:
The `super.commandBus()` call already does the assertion :)